v3/plugins/redhat/rhamt/0.0.8/meta.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: v2
+id: redhat.rhamt-vscode-extension
+version: 0.0.8
+type: VS Code extension
+name: rhamt
+displayName: Red Hat Application Migration Toolkit
+title: Red Hat Application Migration Toolkit
+description: Application Migration and Modernization using RHAMT
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+publisher: redhat
+repository: https://github.com/windup/rhamt-vscode-extension
+category: Other
+firstPublicationDate: "2019-06-19"
+spec:
+  endpoints:
```

Comment:
@johnsteele the [PR check job reported an error](https://circleci.com/gh/eclipse/che-plugin-registry/723?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link):

```
in "v3/plugins/redhat/rhamt/0.0.8/meta.yaml", line 15, column 3
expected <block end>, but found '-'
```

can you please have a look?